conda/plugins/subcommands/doctor/health_checks.py

```diff
@@ -5,22 +5,25 @@
 from __future__ import annotations
 
 import json
+import os
 from logging import getLogger
 from pathlib import Path
-from typing import TYPE_CHECKING
+
+import requests
 
 from ....core.envs_manager import get_user_environments_txt_file
 from ....exceptions import CondaError
 from ....gateways.disk.read import compute_sum
 from ... import CondaHealthCheck, hookimpl
 
-if TYPE_CHECKING:
-    import os
-
 logger = getLogger(__name__)
 
 OK_MARK = "✅"
 X_MARK = "❌"
+CA_BUNDLE_TEST_URL = context.channel_alias.urls()[
+    0
+]  # using one of the channel aliases url ensures that the health check also runs on machines that are on an intranet connection. As in such a case the `context.channel_alias.urls()[0]` might be pointing to an internal url.
+# CA_BUNDLE_TEST_URL = "https://example.com"  # IANA reserved domain (more info: https://www.iana.org/help/example-domains)
```

Comment:
Accessing `context.channel_alias` outside the function may trigger an import-time side effect, which might be a problem since plugins are loaded pretty early in the startup process of conda. Let's move that into the check function below (I'll add a suggestion)